cmd/s3-benchmark/main.go

```diff
@@ -23,11 +23,14 @@ func main() {
 	var conf Config
 
 	flag.StringVar(&conf.Endpoint, "endpoint", "127.0.0.1:7777", "endpoint address")
+	flag.StringVar(&conf.SatelliteAddr, "satelliteaddr", "127.0.0.1:7778", "satellite address")
```

Comment:
Wouldn't this be the same as endpoint? Just having argument as `satellite` would be fine.

Reply:
Also might be confusing having both `endpoint` and `satellite`. Maybe rename `endpoint` to `s3-endpoint` or rename such that the same variable covers both?

Reply:
I was thinking about using `endpoint` for both but it is not the same port. If we have two different options we could call both with the same line.
`s3-benchmark ... -endpoint localhost:7777 -satelliteaddr localhost:7778 -client ...`

Let me rename the options to s3-gateway and satellite.